ingress/controllers/nginx/examples/default/rc-default.yaml

```diff
@@ -14,11 +14,18 @@ spec:
         k8s-app: nginx-ingress-lb
         name: nginx-ingress-lb
     spec:
-      terminationGracePeriodSeconds: 60      
+      terminationGracePeriodSeconds: 60
       containers:
       - image: gcr.io/google_containers/nginx-ingress-controller:0.8.2
         name: nginx-ingress-lb
         imagePullPolicy: Always
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 80
```

Comment:
> Make the default rc.yamls work behind a cloud provider lb by documenting that the rc must serve a health check on :80, either on some un-claimable url (weird) or by proxying to a health check url on the default backend

This is not hard to change. I can add a `/healthz` location in the default server.